client/elements/addons/sc-top-sheet-search-filter.js

```diff
@@ -108,10 +113,10 @@ export class SCTopSheetSearchFilter extends SCTopSheetCommon {
             <tr>
               <td>Filter results by <code>author</code>.</td>
               <td>author:sujato cat<br>
-                  author:sujato Katze<br>
-                  author:sujato Buddha OR Sāvatthī<br>
-                  author:sujato Buddha AND Sāvatthī
-              </td>
+              author:sabbamitta Katze<br>
+              author:sujato Buddha OR Sāvatthī<br>
```

Comment:
Is this example even sensible?  Maybe pick a better example for when you'd want to use OR (or remove it)